src/test/java/gov/usgs/earthquake/origin/OriginIndexerModuleTest.java

```diff
@@ -89,13 +100,83 @@ public void getProductSummary() throws Exception {
     product.setLatitude(latitude);
     product.setLongitude(longitude);
     summary = module.getProductSummary(product);
-    Assert.assertEquals(service.getEventTitle(latitude, longitude), summary.getProperties().get("title"));
+
+    Assert.assertEquals(module.getEventTitle(latitude, longitude), summary.getProperties().get("title"));
+  }
+
+  @Test
+  public void formatEventTitleTest() {
+    int distance = 0;
+    double azimuth = 0.0;
+
+    // OUS location
+    String expectation = String.format("0 km S of name, country_name");
+    JsonObject feature = Json.createObjectBuilder()
+        .add("properties",
+            Json.createObjectBuilder().add("name", "name").add("country_code", "country_code")
+                .add("admin1_name", "admin1_name").add("country_name", "country_name").add("distance", distance)
+                .add("azimuth", azimuth))
+        .build();
+    Assert.assertEquals(expectation, module.formatEventTitle(feature));
+
+    // US location
+    expectation = String.format("0 km S of name, admin1_name");
+    feature = Json.createObjectBuilder()
+        .add("properties",
+            Json.createObjectBuilder().add("name", "name").add("country_code", "us").add("admin1_name", "admin1_name")
```

Comment:
Minor formatting, put each add on its own line and alphabetize unless another order is needed

Reply:
Updated.